### PR TITLE
add key pages api

### PR DESCRIPTION
**Pull request by WillTaylor22**

### CODE REVIEW

File: conf/urls.py [hunk 1]
```diff
@@ -212,12 +212,16 @@
         core.views.ServiceNoLongerAvailableView.as_view(),
         name='custom-page'
     ),
+    url(
+        r'^search/key-pages$',
```
Comment (sdonk) [inline conf/urls.py]: trailing slash missing

File: core/tests/test_views.py [hunk 2]
```diff
@@ -610,6 +610,13 @@ def test_search_view(client):
         assert context['error_status_code'] == 500
 
 
+def test_search_key_pages_view(client):
+    response = client.get(reverse('search-key-pages'))
+    feed_parsed = json.loads(response.content)
+    assert feed_parsed["orderedItems"][0]["object"]["name"] == \
```
Comment (sdonk) [inline core/tests/test_views.py]: please don't mix single and double quotes

File: core/views.py [hunk 3]
```diff
@@ -328,5 +328,12 @@ def get_context_data(self, **kwargs):
                 return helpers.parse_results(response, query, page)
 
 
+class SearchKeyPagesView(TemplateView):
+    """ Returns data on key pages (such as the Get Finance homepage) to
+        include in search that are otherwise not provided via other APIs.
+    """
+    template_name = 'core/search-key-pages.txt'
```
Comment (richtier) [inline core/views.py]: `search-key-pages.txt` looks more like a json file. could you name it `search-key-pages.jsone, please

Comment (richtier) [reply core/views.py]: will services that talk to this endpoint care about the content-type header? if so the default content type will be `text/html` but for json responses it's normally `application/json`

Comment (WillTaylor22) [reply core/views.py]: It doesn't

**Pull request merged**